app/services/beta/search/perform_search_service.rb

```diff
@@ -1,27 +1,24 @@
 require 'tariff_jsonapi_parser'
 require 'errors'
+require 'retriable'
```

Comment:
We use zeitwerk which should mean that you don't need these requires

Reply:
lib isn't autoloaded